docs/concepts/apiserver_admission_plugins.md

```diff
@@ -124,6 +124,16 @@ This admission controller reacts on `CREATE` and `UPDATE` operations for `Shoot`
 It tries to assign a default domain to the `Shoot` if it gets scheduled to a seed that enables DNS for shoots (`.spec.settings.shootDNS.enabled=true`).
 It also validates that the DNS configuration (`.spec.dns`) is not set if the seed disables DNS for shoots.
 
+## `ShootNodeLocalDNSEnabledByDefault`
+
+_(enabled by default)_
```

Comment:
Probably it should be disabled by default for now?

Reply:
According to https://github.com/gardener/gardener/pull/6057#discussion_r891366619, you consider this feature still as alpha:

> [...] as the feature is officially still alpha 

Hence, disabling it by default seems to be reasonable. Operators can still decide to pro-actively enable it for their landscpaes.

Reply:
I've disabled it by default.